<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeIP</class>
 <widget class="QDialog" name="changeIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Please input the IPv4 address of your WiFi Bot</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPlainTextEdit" name="case1">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>192</string>
        </property>
        <property name="tabStopWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="case2">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>168</string>
        </property>
        <property name="tabStopWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="case3">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>1</string>
        </property>
        <property name="tabStopWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="case4">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>106</string>
        </property>
        <property name="tabStopWidth">
         <number>3</number>
        </property>
        <property name="maximumBlockCount">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
